config/prometheusqueries.sample.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "Description": "Request duration",
+        "Query": "rate(mattermost_http_request_duration_seconds_sum[5m])/rate(mattermost_http_request_duration_seconds_count[5m])",
+        "Threshold": 1.0,
+        "Alert": true
+    },
+    {
+        "Description": "Total amount of websockets",
+        "Query": "mattermost_http_websockets_total",
+        "Threshold": 0.0,
+        "Alert": false
+    },
+    {
+        "Description": "Average database store time",
+        "Query": "avg(rate(mattermost_db_store_time_sum[5m]))",
+        "Threshold": 0.05,
+        "Alert": true
+    }
+]
```

Comment:
What about including the queries in the main config file so we avoid having two separate things? Since this is required I'd like to make running this command as straightforward as possible.

Reply:
Done. This also brought a positive side effect where some lines of code were removed, since we don't have to deal with reading and parsing the extra JSON file. 👌